### PR TITLE
Close-loop BZ2217079

### PR DESCRIPTION
**Pull request by lhellebr**

### CODE REVIEW

**Comment by lhellebr** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_remoteexecution.py::TestPullProviderRex::test_positive_run_job_in_chosen_directory

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 5971
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/cli/test_remoteexecution.py::TestPullProviderRex::test_positive_run_job_in_chosen_directory --external-logging
Test Result : ================= 6 passed, 528 warnings in 4274.13s (1:11:14) =================
```

**Comment by lhellebr** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_remoteexecution.py::TestPullProviderRex::test_positive_run_job_in_chosen_directory

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 5982
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/cli/test_remoteexecution.py::TestPullProviderRex::test_positive_run_job_in_chosen_directory --external-logging
Test Result : ================= 6 passed, 521 warnings in 4230.18s (1:10:30) =================
```

**Comment by shweta83** [comment]
@lhellebr Can you please add some description on the PR?

File: tests/foreman/cli/test_remoteexecution.py [hunk 1]
```diff
@@ -1341,6 +1342,110 @@ def test_positive_run_job_on_host_converted_to_pull_provider(
...
+        setting_update,
+    ):
+        """Run job on host registered to mqtt, check it honors run directory
+
+        :id: d4ae37db-d3b6-41b3-bd98-48c29389e4c5
+
+        :expectedresults: Verify the job was successfully ran against the host registered to mqtt, in the correct directory
+
+        :CaseImportance: High
+
```
Comment (pondrejk) [inline tests/foreman/cli/test_remoteexecution.py:1371]: no bz tag?

**Review by pondrejk — APPROVED** [review]
ack, just add bz to docstring

**Review by shubhamsg199 — APPROVED** [review]

**Comment by lhellebr** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_remoteexecution.py::TestPullProviderRex::test_positive_run_job_in_chosen_directory

**Review by pnovotny — APPROVED** [review]

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 6125
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/cli/test_remoteexecution.py::TestPullProviderRex::test_positive_run_job_in_chosen_directory --external-logging
Test Result : ================= 6 passed, 557 warnings in 4459.27s (1:14:19) =================
```

**Comment by pondrejk** [comment]
@lhellebr please rebase, seems there's some conflict to be resolved

**Comment by lhellebr** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_remoteexecution.py::TestPullProviderRex::test_positive_run_job_in_chosen_directory

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 6657
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/cli/test_remoteexecution.py::TestPullProviderRex::test_positive_run_job_in_chosen_directory --external-logging
Test Result : ================= 6 passed, 604 warnings in 4609.76s (1:16:49) =================
```

**Review by pnovotny — APPROVED** [review]

File: tests/foreman/cli/test_remoteexecution.py [hunk 2]
```diff
@@ -977,7 +1082,7 @@ def test_positive_run_job_on_host_registered_to_pull_provider(
     ):
         """Run custom template on host registered to mqtt, check effective user setting
 
-        :id: 759ad51d-eea7-4d7b-b6ee-60af2b814464
+        :id: da9985aa-d319-418f-bc5e-7f4cb4fc4916
```
[the remark this answers is not in the document]
Comment (lhellebr) [reply tests/foreman/cli/test_remoteexecution.py:1084]: No, it's not. It somehow happened during the rebase.

**Comment by lhellebr** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_remoteexecution.py::TestPullProviderRex::test_positive_run_job_in_chosen_directory

**Comment by lhellebr** [comment]
CQ failures unrelated, dependency installation failure

**Comment by lhellebr** [comment]
After PRT passes again, let's please merge it. I don't enjoy rebasing a PR after it's been pending for two months.

**Comment by lhellebr** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_remoteexecution.py::TestPullProviderRex::test_positive_run_job_in_chosen_directory

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 6676
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/cli/test_remoteexecution.py::TestPullProviderRex::test_positive_run_job_in_chosen_directory --external-logging
Test Result : ================= 6 passed, 545 warnings in 4237.17s (1:10:37) =================
```

**Review by jameerpathan111 — APPROVED** [review]

**Pull request merged**